cms/djangoapps/contentstore/views/xblock.py

```diff
@@ -21,7 +22,13 @@ def xblock_resource(request, block_type, uri):  # pylint: disable=unused-argumen
     """
     try:
         xblock_class = XBlock.load_class(block_type, select=settings.XBLOCK_SELECT_FUNCTION)
-        content = xblock_class.open_local_resource(uri)
+        # Note: in debug mode, return any file rather than going through the XBlock which
+        # will only return public files. This allows unbundled files to be served up
```

Comment:
It looks like these methods are identical in the 2 files-- please move to xblock utils.